pkg/scheduler/plugins/priority/priority.go

```diff
@@ -113,6 +113,26 @@ func (pp *priorityPlugin) OnSessionOpen(ssn *framework.Session) {
 	}
 
 	ssn.AddPreemptableFn(pp.Name(), preemptableFn)
+
+	validJobFn := func(obj interface{}) *api.ValidateResult {
```

Comment:
If there is not special things to do in validJobFn. It's better to remove this function.

Reply:
> If there is not special things to do in validJobFn. It's better to remove this function.

I also agree to delete it.